test/JDBC/expected/BABEL-3172-vu-verify.out

```diff
@@ -0,0 +1,58 @@
+USE babel_3172_test_db;
+GO
+
+
+
+DECLARE @table_id INT
+SET @table_id = (SELECT OBJECT_ID('sys_column_name_vu_t_column_name'));
+DECLARE @invalid_table_id INT
+SET @invalid_table_id = (SELECT OBJECT_ID('sys_column_name_vu_t_column_name_invalid'));
+SELECT * FROM sys.COL_NAME(@table_id, 1);
+SELECT * FROM sys.COL_NAME(@table_id, 3);
+SELECT * FROM sys.COL_NAME(@invalid_table_id, 1);
+SELECT * FROM sys.COL_NAME(@table_id, NULL);
+SELECT * FROM sys.COL_NAME(NULL, 1);
+SELECT * FROM sys.COL_NAME(@table_id, -1);
+SELECT * FROM sys.COL_NAME(-1, 1);
+SELECT * FROM sys.COL_NAME(-1, -1);
```

Comment:
Please add different test case which  include invalid inputs like strings,varbinary etc and also try some testcase where input is an expression and other such things.